src/analytics/addPageAction.ts

```diff
@@ -51,6 +51,10 @@ export const addPageAction = <A extends ActionType, P extends Properties>(
   };
 
   if (typeof newrelic !== "object") {
+    // These will only print when new relic is not available such as during local development
+    console.log("____ ANALYTICS EVENT ____");
```

Comment:
can we put these all on one line? console.log("ANALYTICS EVENT:", {name, attributesToSend})